app/models/feature.rb

```diff
@@ -0,0 +1,8 @@
+class Feature < ApplicationRecord
+  has_and_belongs_to_many :plans
+
+  validates :name, presence: true, length: { maximum: 50 }
+  validates :code, presence: true, length: { maximum: 50 }
```

Comment:
I think this code should only have alphanumeric characters! What is your say on the matter?

Reply:
It didn't mentioned in requirements. It could be anything.